feeds/wlan-ap/apc/src/src/ubus.c

```diff
@@ -159,10 +192,10 @@ ubus_init(void) {
 #endif
 	add_object(&apc_object);
 	notify_timer = tm_new_set(apc_send_notification, NULL,
-				  0, APC_NOTIFY_INTERVAL);
+				  0, 1);
```

Comment:
Why not change APC_NOTIFY INTERVAL to 1?  It's defined in this file and not used except in line 162 and 165.

Reply:
Added new macro APC_NOTIFY_CHECK 1